app/commands/put_content_with_links.rb

```diff
@@ -10,10 +10,20 @@ def call(downstream: true)
       PathReservation.reserve_base_path!(base_path, content_item[:publishing_app])
 
       if downstream
-        Adapters::DraftContentStore.call(base_path, content_item_for_content_store)
-        Adapters::ContentStore.call(base_path, content_item_for_content_store)
-
-        PublishingAPI.service(:queue_publisher).send_message(content_item_for_message_bus)
+        content_store_payload = Presenters::ContentStorePresenter::V1.present(
+          content_item,
+          access_limited: false,
+          update_type: false
+        )
+
+        Adapters::DraftContentStore.call(base_path, content_store_payload)
+        Adapters::ContentStore.call(base_path, content_store_payload)
+
+        message_bus_payload = Presenters::ContentStorePresenter::V1.present(
+          payload,
+          access_limited: false,
```

Comment:
Feels to me that having different presenters for each downstream representation would be clearer, even if they just wrap some shared functionality.  For example, removing access limiting and update type are used here, but the removal of the base path is left to the command to do.  Encapsulating all that behaviour would make life easier for this and for reuse around the app.